airflow/providers/amazon/aws/hooks/redshift_cluster.py

```diff
@@ -183,3 +187,86 @@ def get_cluster_snapshot_status(self, snapshot_identifier: str, cluster_identifi
             return snapshot_status
         except self.get_conn().exceptions.ClusterSnapshotNotFoundFault:
             return None
+
+
+class RedshiftHookAsync(AwsBaseHookAsync):
+    """Interact with AWS Redshift using aiobotocore library"""
+
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        kwargs["client_type"] = "redshift"
+        kwargs["resource_type"] = "redshift"
```

Comment:
This is redundant because `resource_type` it only could be created by `boto3`.
And also `redshift` is not listed as a valid resource in `boto3`

```python
Python 3.9.10 (main, Feb 25 2022, 16:54:01) 
Type 'copyright', 'credits' or 'license' for more information
IPython 8.7.0 -- An enhanced Interactive Python. Type '?' for help.
PyDev console: using IPython 8.7.0
Python 3.9.10 (main, Feb 25 2022, 16:54:01) 
[Clang 13.0.0 (clang-1300.0.29.30)] on darwin

import boto3
boto3.__version__
Out[3]: '1.26.43'

session = boto3.session(region_name="us-east-1")  # All features become first in N.Virginia
session.get_available_resources()

Out[6]: 
['cloudformation',
 'cloudwatch',
 'dynamodb',
 'ec2',
 'glacier',
 'iam',
 'opsworks',
 's3',
 'sns',
 'sqs']
```